pages/guide.md

```diff
@@ -33,6 +33,7 @@ Answers to other common compliance questions appear below.
 * [What is the scope of BOD 18-01?](#what-is-the-scope-of-bod-18-01)
 * [What is a second-level domain?](#what-is-a-second-level-domain)
 * [How does the web security requirement in BOD 18-01 differ from M-15-13?](#how-does-the-web-security-requirement-in-bod-18-01-differ-from-m-15-13)
+* [My device is HTTPS already and doesn't natively support HSTS. Can DHS mark this a false positive?](#my-device-is-https-already-and-doesnt-natively-support-hsts-can-dhs-mark-this-a-false-positive)
```

Comment:
There's a typo here (missing `as` in `Can DHS mark this as a false positive?`), and I would rephrase a bit more precisely as:

> My device supports HTTPS already, but doesn't natively support HSTS. Can DHS mark this as a false positive?